### DISCUSSION TITLE
Render VTK in a smaller window

### DISCUSSION THREAD

**Discussion by pazars** [Kitware/trame#85] [Trame/VTK] [answered]
After following the tutorial, I tried to setup 2 Vuetify containers where one renders the VTK cone (from example) and the other does something else. Basically, have VTK on one side of the screen and something else on the other. However, it was not working. Nothing was rendering, not even the header, which was not modified. 

I went back to the example and tried isolating the issue. Turns out VTK won't render if its parent container does not inherit the fill-height class. Is this intended or is this a bug?

To reproduce: _trame-tutorial/03_html/solution_final.py_ with:

`        with vuetify.VContainer(
            fluid=True,
            classes="pa-0"
        )`

**Comment by jourdain** [comment, marked as answer]
This is the expected behavior. 

Basically to understand the concept, you need to think in the same way as a window on a house or a building. The size of the window is not affected by the size of the world out there but by the layout of the facade or the size set to that window. For the vtk window/view, it is the same. The layout will either provide room for its content or not... 

In your case, your layout was not imposing any size (no fill-height) to your window and you did not fix it either (`style="width: 200px; height: 500px;"`). Therefore the window has a height of 0 which made it "invisible".

HTH
